README.md

```diff
@@ -124,9 +124,16 @@ Agent should drop the whole message and return an error to the client. In additi
 message MAY contain additional data (such as `Span`s). As long as it has a valid identifier
 assoicated, Agent should handle the data properly, as if they were sent in a subsequent message.
 Identifier is no longer needed once the streams are established.
-3. If streams were disconnected and retries failed, the Library identifier would be considered
-expired on Agent side. Library needs to start a new connection with a unique identifier
-(MAY be different than the previous one).
+3. On Library side, if connection to Agent failed, Library should retry indefintely. (Reason:
```

Comment:
> indefinitely

Subject to available/configured memory buffer size.

Reply:
Will send a follow-up PR to update.